pyserini/eval/trec_eval.py

```diff
@@ -36,13 +36,14 @@
 import subprocess
 import sys
 import tempfile
+from typing import Any
 
 import jnius_config
 import pandas as pd
 
 # Don't use the jdk.incubator.vector module.
-jar_directory = str(importlib.resources.files("pyserini.resources.jars").joinpath(''))
-jar_path = glob.glob(os.path.join(jar_directory, '*.jar'))[0]
+jar_directory = str(importlib.resources.files("pyserini.resources.jars").joinpath(""))
```

Comment:
can we revert back to single quotes?

Reply:
just the quotes or ignore black entirely?

Reply:
just the quotes so I can more clearly see the substantive code additions?

Reply:
btw, can we add a test case?